eth/gasprice/gasprice_test.go

```diff
@@ -208,7 +208,6 @@ func TestSuggestTipCap(t *testing.T) {
 		fork   *big.Int // London fork number
 		expect *big.Int // Expected gasprice suggestion
 	}{
-		{nil, big.NewInt(params.GWei * int64(30))},
```

Comment:
can you elaborate why prefer removing this?

Reply:
I find another way to fix this test: https://github.com/scroll-tech/go-ethereum/pull/938/commits/468348c5f264cf1cf1a105844420ada121454be1